data/scripts/talkactions/ban.lua

```diff
@@ -0,0 +1,133 @@
+local ban = TalkAction("/ban")
+
+function ban.onSay(player, words, param)
+	if not player:getGroup():getAccess() then
```

Comment:
remove this check and use

```ban:access(true)```

between the separator and register methods, same with the others